_docs/installation.md

```diff
@@ -92,11 +92,20 @@ For the first run, get access to the shell within the Docker container
 $ docker run -it -v /mnt/dropbox:/dropbox maestraldbx/maestral:latest ash
 ```
 
-where `/mnt/dropbox` is the directory which contains the `Dropbox` directory.
-Maestral runs with `UID` 1000, make sure that the user owns `/mnt/dropbox` and the
-contents within (`chown -R 1000 /mnt/dropbox`).
+where `/mnt/dropbox` is the directory which contains the `Dropbox` directory and
+Maestral's configuration. Maestral runs with `UID` 1000, make sure that the user
+owns `/mnt/dropbox` and the contents within (`chown -R 1000 /mnt/dropbox`).
 
-Later, if you want just a `maestral start`, just execute
+Then run Maestral's setup
+
+```console
+$ maestral start
+```
+
+This will prompt you to authorize Maestral to access your Dropbox account and
+establish your Dropbox folder at `/dropbox/Maestral`.
```

Comment:
IIRC, the default folder name will be `/Dropbox/Dropbox (Maestral)`.

Reply:
Hmm, wouldn't it be `/dropbox/Dropbox (Maestral)` in the container?

Reply:
Yes, damn autocorrect. `/dropbox/Dropbox (Maestral)` would be the default location inside the container.